.github/dependabot.yml

```diff
@@ -38,6 +35,6 @@ updates:
       # Pin mockito to 4.x
       - dependency-name: "org.mockito"
         versions: "[5.0.0,)"
-      # Pin byte-buddy to 1.12.x aline with mockito
+      # Pin byte-buddy to 1.12.x align with mockito
```

Comment:
This is minor but unrelated change and perhaps can be avoided in this PR.
Let's see what other's think.

Reply:
+1 for @mystic-lama 's comment. Please recover this line.

Reply:
I removed it, @cxzl25 .

Reply:
Thanks. @dongjoon-hyun